File: 01_Basico/03 - Lacos_Repeticao/Respsota_em_Python/questao10.py
qtd_pessoa = int(input("Quantas pessoas tem na turma: "))
soma_idades = 0
pessoa = 1
for i in range(1,qtd_pessoa + 1):
    idade = int(input(f"Digite a idade da {pessoa}º pessoa: "))
    soma_idades += idade
    pessoa += 1

media = soma_idades / qtd_pessoa
if media >=0 and media <= 25:
    turma = "jovem"
elif media > 25 and media <= 60:
    turma = "adulta"
else:
    turma = "idosa"

print(f"A media de idade é {media}, por tanto a turma é {turma}")

File: 01_Basico/03 - Lacos_Repeticao/Resposta_em_Java_Script/questao10.js
let qtd_pessoa = parseInt(prompt("Quantas pessoas tem na turma: "))
let soma_idades = 0
var pessoa = 1

for (i= 0; i<= qtd_pessoa; 1) {
    let idade = parseInt(prompt(`Digite a idade da ${pessoa}º pessoa: `))
    soma_idades += idade
    i++
    pessoa++
}

var media = soma_idades / qtd_pessoa

if (media >=0 && media <= 25) {
    turma = "jovem"
}else if(media > 25 && media <= 60){
    turma = "adulta"
}else{
    turma = "idosa"
}
console.log(`A media de idade é ${media}, por tanto a turma é ${turma}`)